python-backend/app/api/applications/resources/apllication.py

```diff
@@ -0,0 +1,91 @@
+from flask_restplus import Resource, reqparse
+from flask import request
+from datetime import datetime
+
+from app.extensions import api
+from ..models.application import Application
+from ...mines.mine.models.mine import Mine
+from app.api.utils.resources_mixins import UserMixin, ErrorMixin
+from app.api.utils.access_decorators import requires_role_mine_view, requires_role_mine_create
+
+
+class ApplicationResource(Resource, UserMixin, ErrorMixin):
+    parser = reqparse.RequestParser()
+    parser.add_argument('application_no', type=str, help='Number of the application being added.')
+    parser.add_argument('mine_guid', type=str, help='guid of the mine.')
+    parser.add_argument(
+        'application_status_code',
+        type=str,
+        help='Status of the application being added.',
+        store_missing=False)
+    parser.add_argument(
+        'received_date', type=lambda x: datetime.strptime(x, '%Y-%m-%d') if x else None)
```

Comment:
should this have a store_missing flag

Reply:
No, The only time that is entered and used is on the creation of the application.